commands/api-gateway/route/upstreams/remove.go

```
@@ -0,0 +1,75 @@
+package upstreams
+
+import (
+	"context"
+	"fmt"
+	"github.com/ionos-cloud/ionosctl/v6/commands/api-gateway/completer"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/sdk-go-bundle/products/apigateway/v2"
+	"github.com/spf13/viper"
+)
+
+func RemovetCmd() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
+		Namespace: "apigateway",
+		Resource:  "upstreams",
+		Verb:      "remove",
+		Aliases:   []string{"r"},
+		ShortDesc: "Upstreams consist of schme, loadbalancer, host, port and weight",
+		Example:   "ionosctl apigateway route upstreams remove --gateway-id ID --route-id ID_ROUTE --upstream-id UPSTREAMID",
+		PreCmdRun: func(c *core.PreCommandConfig) error {
+			if err := core.CheckRequiredFlags(c.Command, c.NS, constants.FlagGatewayID, constants.FlagGatewayRouteID, constants.FlagUpstreamId); err != nil {
+				return err
+			}
+
+			return nil
+		},
+		CmdRun: func(c *core.CommandConfig) error {
+			apigatewayId := viper.GetString(core.GetFlagName(c.NS, constants.FlagGatewayID))
+			routeId := viper.GetString(core.GetFlagName(c.NS, constants.FlagGatewayRouteID))
+			upstreamId := viper.GetInt(core.GetFlagName(c.NS, constants.FlagUpstreamId))
+			usedRoutes, _, _ := client.Must().Apigateway.RoutesApi.ApigatewaysRoutesFindById(context.Background(), apigatewayId, routeId).Execute()
+			input := usedRoutes.Properties
+			if input.Upstreams == nil {
+				fmt.Errorf("There are no upstreams defined in this route!")
+			} else if upstreamId == 0 {
+				input.Upstreams = input.Upstreams[1:]
+			} else if upstreamId == 1 {
+				input.Upstreams = append(input.Upstreams[:upstreamId], input.Upstreams[upstreamId+1:]...)
+			} else if upstreamId == 2 {
+				input.Upstreams = input.Upstreams[:len(input.Upstreams)-1]
+			}
```

same as customDomains comment. Error is not returned here either